docs/grid_model/network_subnetwork.md

```diff
@@ -95,6 +95,35 @@ When defining the model, the user has to specify how the different equipment con
 - [Identifiable Short-Circuit](extensions.md#identifiable-short-circuit)
 - [Slack Terminal](extensions.md#slack-terminal)
 
+## Area
+[![Javadoc](https://img.shields.io/badge/-javadoc-blue.svg)](https://javadoc.io/doc/com.powsybl/powsybl-core/latest/com/powsybl/iidm/network/Area.html)
+
+An Area is a geographical zone of a given type.
+
+An Area is composed of a collection of [voltage levels](#voltage-level), and a collection of area boundaries.
+Area boundaries can be terminals of equipments or `Boundary` objects from [dangling lines](#dangling-line).
+
+The area type is used to distinguish between various area concepts of different granularity. For instance: control areas, bidding zones, countries...
+
+A [voltage level](#voltage-level) can belong to several areas, as long as all areas are of different type.
+
+The area boundaries define how interchange are to be calculated for the area.  
+Area interchange is calculated by summing the active power flows across the area boundaries and can be obtained for AC part only (considering only AC boundaries),
+for DC part only (considering only DC boundaries) and in total (AC+DC).  
+Note that if the Area has no boundary explicitly defined, the interchange is considered 0 MW.
+
+For area types that are meant to be used for area interchange control, e.g. in Load Flow simulations, the interchange target of the area can be specified as an input for the simulation.
+Note that this target interchange is for only the AC part of the interchange.
+
+All area interchange values use the load sign convention: positive values indicate that the area is importing, negative values that the area is exporting.
+
+**Characteristics**
+
+| Attribute             | Unit | Description                                                    |
+|-----------------------|------|----------------------------------------------------------------|
+| $AreaType$            |      | To specify the type of Area (eg. ControlArea, BiddingZone ...) |
+| $AcInterchangeTarget$ | MW   | Target AC active power interchange                             |
```

Comment:
My last remark ;-) Here, even if it is not always done in other sections, I would like to know that I can have the list of voltage levels and the list of boundaries.

Reply:
Thank you for the review! This has been modified in 74a3b3d4d2d17f8644e4943e25f4483a861a929d

Reply:
Note: Damien suggested we add a second table with the AreaBoundary attributes to help to understand the AreaBoundaries attribute. It has been done in 5f417e15b72a847d5b9afc8508af8e3837e4aeb6